internal/handlers/common/handler.go

```diff
@@ -17,11 +17,16 @@ package common
 import (
 	"context"
 
+	"github.com/prometheus/client_golang/prometheus"
+
 	"github.com/FerretDB/FerretDB/internal/wire"
 )
 
 // Handler interface represents common commands handlers.
 type Handler interface {
+	// Handle handles the message.
+	Handle(context.Context, *wire.MsgHeader, wire.MsgBody) (*wire.MsgHeader, wire.MsgBody, bool)
```

Comment:
No, that method should not be here. Previously, we discussed if our common interface should have this one method or all methods for each individual command, and decided to stick with the latter. We should not have both, and we should force each handler to implement that quite complicated method.

Reply:
extracted a common router